tests/integration/sdk_api/alerts/test_alerts.py

```diff
@@ -0,0 +1,538 @@
+# Copyright 2023 Iguazio
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import pydantic.error_wrappers
+import pytest
+
+import mlrun
+import mlrun.common.schemas
+import mlrun.common.schemas.alert as alert_constants
+import mlrun.utils
+import tests.integration.sdk_api.base
+from mlrun.utils import logger
+
+
+class TestAlerts(tests.integration.sdk_api.base.TestMLRunIntegration):
+    def test_alert_operations(self):
```

Comment:
Why not split this to smaller tests?

Reply:
integ tests are long to setup/teardown. the reasoning was less runtime